views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+<div class="container" id="about">
+  <div class="page-section">
+    <div class="row">
+      <div class="column-one-quarter">
+         <%= erb :about_nav_menu %>
+      </div>
+      <div class="column-three-quarters">
+        <h1>What’s in EveryPolitician’s data?</h1>
+        <p class="section-intro">
+          Short answer: names. And dates of birth, twitter handles, politicial
+          group memberships, email addresses, honorific titles, image URLs, and
+          all sorts of useful things — all provided we can find them. How
+          complete this data is depends on the sources we’ve found for each
+          country.
+        </p>
+        <p>
+          This is a non-technical overview. To properly understand what
```

Comment:
You say this is the non-technical overview, but it then immediately starts getting quite technical. The description here of how it all hangs together is good, but it seems likely to immediately be off-putting to a non-tech person just wanting to know whether it's even worth looking into using the data (or having a more technical person look into it)